src/_pylibmcmodule.c

```diff
@@ -2491,9 +2554,14 @@ static void _make_excs(PyObject *module) {
     PylibMCExc_Error = PyErr_NewException(
             "pylibmc.Error", NULL, NULL);
 
+    PylibMCExc_CacheMiss = PyErr_NewException(
```

Comment:
Sorry, "root exception" in terms of being a public member, or in terms of its place in the inheritance hierarchy?